sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/polling/Error.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.management.implementation.polling;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * Type holding error details of long-running-operation (LRO) or polling-operation of an LRO.
+ */
+class Error {
```

Comment:
Should we use a different class name? This might be a bit too easy to be mistaken with `java.lang.Error`.

Not a big issue since it is not public.

Reply:
I agree - this name should be more specific. `PollingError` or similar.